migrations/20200901-2-theme/update.sql

```diff
@@ -0,0 +1 @@
+UPDATE modules_settings SET modules_settings.value = 's:9:"Bootstrap";' WHERE modules_settings.module = 'Core' AND modules_settings.name = 'theme';
```

Comment:
Ik vermoed dat dit een test is, en dus weg gaat uit de definitieve PR?

Reply:
ja, test migrations zijn weer allemaal verwijderd